goblhtml.go

```diff
@@ -103,9 +103,17 @@ func WithLayout(l layout.Code) Option {
 	}
 }
 
+// WithState indicates that the document is in a specific state
+// and should be rendered accordingly.
+func WithState(state string) Option {
```

Comment:
Looking at this now, its only actually used for the "void" flag. Would it not make more sense to just call it `WithVoid(true)`?

Now that its not passed through all the underlying methods, its much easier to add new variables in the future, so "State" feels too generic.

Reply:
I personally thought `WithState` made sense, since I don't rule out supporting other statuses (like draft) down the line. In that case, having a single option would likely make life easier for the caller.

That said, I don’t want to get stuck in bike-shedding territory, so I’ve changed it to `WithVoid` in #31cbb70 😅